opc-ua-stack/stack-core/src/main/java/org/eclipse/milo/opcua/stack/core/serialization/xml/XmlDecoder.java

```diff
@@ -734,4 +753,15 @@ private String readCharacterContent() throws UaSerializationException {
         }
     }
 
+    private String readElementText(String replacement) throws UaSerializationException {
```

Comment:
I think this method would be better expressed if implemented to this signature: `private Optional<String> readElementText() throws UaSerializationException`.

Callers could then use `Optional#orElse` to handle the replacement scenario.